### DISCUSSION TITLE
Earn more "miles", or decrease item cost

### DISCUSSION THREAD

**Discussion by OctopusButtons** [emoose/OutRun2006Tweaks#214] [Suggestions]
### Describe the feature you'd like to see added

Because the earning of "Miles" (points used to "buy" music, cars, etc, in the "Showroom" shop) is so painfully slow and miserly of the developers, if the numbers could be tweaked to make it more generous that would be pleasant and helpful.  I looked through the ini file but don't see existing parameters for it.  If you were able to finagle the back-end memory addresses/values and then have an ini parameter for Generous Credits/Prices, either increased Miles after racing or decreased cost in the shop would be helpful.

Or if anyone can tell me a way to tweak those numbers myself, I'll document it and do a pull request for that added info in the readme.

**Comment by Tenome** [comment]
If you want to cheat the miles, just use the code MILESANDMILES as your license name. Every time you enter it, you'll get 1000000 miles.

**Reply by OctopusButtons** [reply]
Oh I finally got the MILESANDMILES cheat code to work.

* Edit your existing license by pressing F2 and then Edit License
* Enter the name MILESANDMILES
* Click done on the **red** in-game to confirm the name.  Clicking that done button will bring you to another done button.
* **DO NOT** CLICK "DONE" to confirm finalizing the license.  Yes, there are two different "DONE" buttons, you should only click the red one on the in-game keyboard.
* PRESS ESCAPE instead of done, meaning you're cancelling the procedure without saving the license card
* You'll now have a bunch of thousands of miles

I had tried a few variations of that process and it wasn't working, and I also bad info online " MILESANDMILES" [sic] with a space in front which seems wrong.  Steps above work.

**Comment by emoose** [comment]
The miles didn't really seem too bad for me, by the end of the C2C missions I think I was able to buy majority of the store there (then again the hundreds of tests with OR / HA modes might have tipped the scales a bit :P)

I guess for people that just want to play every so often and not grind the whole C2C missions some kind of scaling/difficulty option might be useful though, either scaling the amount awarded or maybe even adjusting the prices in the shop.

**Comment by OctopusButtons** [comment]
Thank you by the way, you have resurrected a perished classic, it's really great.  THANK YOU for your work on the mod and for giving it to everyone.
